packages/commonwealth/client/scripts/controllers/server/roles.ts

```diff
@@ -14,7 +14,7 @@ import {
   PermissionManager,
   ToCheck,
   everyonePermissions,
-} from 'commonwealth/server/util/permissions';
+} from 'commonwealth/shared/permissions';
```

Comment:
seems like we also do just "shared/*" elsewhere in the repo. is the commonwealth prefix needed here?

Reply:
it's not needed no, we can do relative import instead